lib/index.ts

```diff
@@ -235,9 +235,10 @@ const removeOmitted = (swaggerFieldSchema: {
 function documentModel(Model, options: { props?: string[], omitId?: boolean } = {}): any {
   let {
     props = [],
+    omitId = true,
```

Comment:
The API would be a lot more flexible if it took an array of fields to omit instead of a boolean for a specific field. Would you mind changing the implementation to receive omit as an array of strings of keys to omit?